src/multiSig/errors/invalid.multiSig.address.error.ts

```diff
@@ -0,0 +1,5 @@
+export class InvalidMultiSigAddressError extends Error {
+    constructor(message: string) {
+        super(message);
+    }
+}
```

Comment:
1. Let's move this to `multiSig.errors.ts` in the root of the `multiSig` folder to be consistent with other errors in the code base _(see `src/framework/dependencies.errors.ts`)_.

2. Let's rename this to a more generic `MultiSigError`, and we can poly-use it like so:
```typescript
// For Invalid Addresses
throw new MultiSigError('Invalid MultiSig Address');

// For Other MultiSig error
throw new MultiSigError('Gnosis Backend Unreachable');

// etc ...
```

3. Let's override the `name` of the `Error`:
```typescript
export class MultiSigError extends Error {
    constructor(message: string) {
        super(message);
        this.name = 'MultiSigError';
    }
}
```

Reply:
got it. I used integration folder as a base, that's why I decided to add separate folder